README.md

```diff
@@ -1 +1,106 @@
-# vacations-booking-back-end
+![](https://img.shields.io/badge/microverse-blueviolet)
+
+# Vacations booking back-end
+
+> "Vacations booking back-end" project is a Rails API application which provide the API endpoints for creating/removing different destinations by admin users and creating reservations by other logged users.
+
+![image](https://user-images.githubusercontent.com/63932912/143224920-5cc44446-f6f1-47ee-b737-78e24e5613f7.png)
```

Comment:
Try to make it clear that you used swagger documentation and just a bit of industry advise: Swagger is known not only as an excellent documentation tool but has the ability for someone to test API make it clear if they can use it for this as well.

Give more detail on endpoints whether you mention swagger documentation or give overview here or both,